README.md

```diff
@@ -66,141 +66,27 @@ at https://www.azureiotsuite.com.
 Getting Started
 ===============
 
-## Prerequisites
-* [Azure Subscription](https://azure.microsoft.com/free/) (also see [permissions guidelines](https://docs.microsoft.com/azure/iot-suite/iot-suite-permissions))
-
-## Setup
-1. Open a terminal window or command shell
-1. Check out the repository and submodules
-    ```
-    git clone --recursive https://github.com/Azure/azure-iot-pcs-remote-monitoring-java
-    ```
-    If you cloned the repo without the `--recursive` flag and it's your first time updating the submodules, you can use
-    `git submodule update --init --recursive`. More information on submodules [here](https://git-scm.com/book/en/v2/Git-Tools-    Submodules). 
```

Comment:
Users reaching the solution in GitHub typically want to work directly with the code, so this seems useful information, no?